### DISCUSSION TITLE
Is it possible to change color with CDN?

### DISCUSSION THREAD

**Discussion by devkennyy** [simple-icons/simple-icons#7634] [Help with SVGs] [answered]
<img height="32" width="32" src="https://cdn.jsdelivr.net/npm/simple-icons@v7/icons/discord.svg" />

Would it be possible to change the colour when referencing an icon like this?

**Comment by sachinraja** [comment, marked as answer]
See https://github.com/simple-icons/simple-icons/discussions/6895
